common/src/main/java/org/jboss/hal/testsuite/creaper/command/RemoveSocketBinding.java

```diff
@@ -44,7 +45,7 @@ public void apply(OnlineCommandContext ctx) throws CommandFailedException {
                 .and("socket-binding", socketBindingName);
 
         try {
-            operations.remove(socketBindingAddress);
+            operations.removeIfExists(socketBindingAddress);
```

Comment:
This is a little bit against concept of creapers remove commands in creaper itself. They would fail if there would be nothing to remove.